env/HERA.env

```diff
@@ -60,7 +60,7 @@ elif [[ "${step}" = "prep_emissions" ]]; then
 
     export APRUN="${APRUN_default}"
 
-elif [[ "${step}" = "waveinit" ]] || [[ "${step}" = "waveprep" ]] || [[ "${step}" = "wavepostsbs" ]] || [[ "${step}" = "wavepostbndpnt" ]] || [[ "${step}" = "wavepostbndpntbll" ]] || [[ "${step}" = "wavepostpnt" ]]; then
+elif [[ "${step}" = "waveinit" ]] || [[ "${step}" = "waveprep" ]] || [[ "${step}" = "wavepostsbs" ]] || [[ "${step}" = "wavepostbndpnt" ]] || [[ "${step}" = "wavepostbndpntbll" ]] || [[ "${step}" = "wavepostpnt" ]]|| [[ "${step}" = "wavestat" ]] ; then
```

Comment:
I'd recommend making this change on any env/*.env file that has wave jobs in it.